src/status_im/ui/screens/about_app/styles.cljs

```diff
@@ -0,0 +1,21 @@
+(ns status-im.ui.screens.about-app.styles
+  (:require [status-im.ui.components.colors :as colors]))
+
+(def peer-view
+  {:background-color colors/white
+   :padding-bottom 5
+   :padding-horizontal 16})
+
+(def peer-text
+  {:font-size 10
+   :color colors/black})
+
+(def about-title
+  {:background-color colors/white
+   :font-size 20
```

Comment:
redundant?

Reply:
It looks like, it displayed a gray background without it

Reply:
I mean the font-size

Reply:
oh could be :) let me  check